iree/test/micro_benchmarks/run_benchmark_on_android.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+MLIR_FILE_PATH=${1}
+ARGS=${@:2}
+
+adb shell 'mkdir /data/local/tmp'
```

Comment:
If we are checking this in, it needs more documentation.

Reply:
yeah, this would probably be better as a gist - otherwise it becomes yet another thing to support.

Reply:
We have something similar in python scripts.

https://github.com/google/iree/blob/main/build_tools/mako/compile_android_modules.py
https://github.com/google/iree/blob/main/build_tools/mako/benchmark_modules_on_android.py

The targets are configured in https://github.com/google/iree/blob/main/build_tools/mako/configuration.py

We can probably reuse them.

Reply:
I remembered that I dropped comments about scripts somewhere, but it seems gone...

We have something similar in build_tools/mako/ which is mainly for mako benchmarks. We probably can reuse some of codes from there.

https://github.com/google/iree/blob/main/build_tools/mako/compile_android_modules.py
https://github.com/google/iree/blob/main/build_tools/mako/benchmark_modules_on_android.py

They are configured in https://github.com/google/iree/blob/main/build_tools/mako/configuration.py
I think we can add MICROBENCHMARKS, I'm happy to help on that once this gets check-in.